pkgs/data/fonts/google-fonts/default.nix

```diff
@@ -18,10 +20,6 @@ stdenv.mkDerivation rec {
     chmod -x $dest/*.ttf
   '';
 
-  outputHashAlgo = "sha256";
-  outputHashMode = "recursive";
-  outputHash = "0q03gg0sh2mljlbmhamnxz28d13znh9dzca84p554s7pwg6z4wca";
```

Comment:
Yep, that was also my understanding

On Tue, Sep 13, 2016, 07:09 Franz Pletz <notifications@github.com> wrote:

> In pkgs/data/fonts/google-fonts/default.nix
> <https://github.com/NixOS/nixpkgs/pull/18499#discussion_r78497280>:
>
> > @@ -18,10 +20,6 @@ stdenv.mkDerivation rec {
> >      chmod -x $dest/*.ttf
> >    '';
> >
> > -  outputHashAlgo = "sha256";
> > -  outputHashMode = "recursive";
> > -  outputHash = "0q03gg0sh2mljlbmhamnxz28d13znh9dzca84p554s7pwg6z4wca";
>
> I think the fetchFromGitHub should fix the build failures as Github
> regenerates archives with different timestamps.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/18499/files/1723b961b06ece08a376f75f529a2bfceedce2c5#r78497280>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAjvSw3xYxTOcm0pVIOgj7b0T0IXqHBQks5qpi_8gaJpZM4J51-L>
> .
>